### ISSUE TITLE
[Gallery][iOS/Android] Overview - At the bottom of the page 'Share' link is not working

### ISSUE THREAD

**Issue by sakshi173** [unoplatform/uno#9012]
### Current behavior

![Share link](https://user-images.githubusercontent.com/74563602/173095547-d84bc285-45d3-4702-98e0-ac9ba280dc55.png)

### Expected behavior

when click/tap on the share link the page should be available to share.

### How to reproduce it (as minimally and precisely as possible)

1. Open Uno Gallery APP
2. Go to the 'Overview' Page.
3. Scroll down to the 'Overview' Page.
4. Go to the 'Share' link at the bottom of the page.
5. Tap/click on the 'Share' link.
6. Observe that nothing is happening.

### Affected platforms

iOS, Android

**Comment by sakshi173** [comment]
Still this bug is reproducible, due to that opening it.

**Comment by agneszitte-nventive** [comment]
@sakshi173 is this issue still reproducible lately please?

**Comment by sakshi173** [comment]
@agneszitte-nventive Yes this issue is still present. Please look into it.

**Comment by kazo0** [comment]
@sakshi173 you're seeing this on mobile devices or on a web browser?

**Comment by sakshi173** [comment]
@rajamatt and @kazo0 , I am seeing in mobile device and when clicking on share link nothing is happening.

https://github.com/unoplatform/uno/assets/74563602/01c5ed70-6d7e-4a04-a250-e3ce8f41bc69

**Comment by MartinZikmund** [comment]
@sakshi173 Which iOS version is this? And does it still repro on Android too?

**Comment by sakshi173** [comment]
@MartinZikmund , Yes Still this issue is present for both Android and iOS.
I am checking for iOS version 15.6.1 and Model is iphone 13 pro Max.

**Comment by Youssef1313** [comment]
Looking at the code, it's expected this is happening:

https://github.com/unoplatform/Uno.Gallery/blob/9fb342c26e0648332eb0adfb62dd2be0b343d950/Uno.Gallery/Uno.Gallery.Shared/Controls/SamplePageLayout.cs#L167-L173

It only works for `!NET6_0_OR_GREATER`, which means it used to work only for Xamarin targets which are gone now.

**Comment by Youssef1313** [comment]
I'm looking into it

**Comment by Arieldelossantos** [comment]
yes, i've noticed too that it was removed on .Net6 upgrade... i was trying to find a workaround based on this documentation from Uno... [Sharing Content](https://platform.uno/docs/articles/features/windows-applicationmodel-datatransfer.html)

**Comment by Youssef1313** [comment]
@Arieldelossantos Oh I didn't know about that!
I think your approach could work better. I'll mark my PR as a draft then.
